docs/pages/guides/using-push-notifications-services.mdx

```diff
@@ -0,0 +1,106 @@
+
+## Expo push notifications
+
+[Expo Notifications](/versions/latest/sdk/notifications/) provides a unified API for handling push notifications across Android and iOS. It integrates seamlessly with your Expo account and is free to use.
+
+### Key capabilities
+
+- Fully compatible with the [`expo-notifications`](/versions/latest/sdk/notifications/) library
+- Includes an Expo Dashboard to track notifications and user engagement
+- Supports testing notifications with the [Expo Notifications Tool](https://expo.dev/notifications)
+
+### Considerations and limitations
+
+- Notification extensions are not built-in, but you can set them up manually following Apple's guidelines.
+- Limited to 600 notifications per second per project, which works for most apps but may not suit high-volume scenarios
+
+For implementation details, see the following guides:
+
+<BoxLink
+  title="Expo push notifications overview"
+  description="Learn more about Expo push notifications."
+  href="/push-notifications/overview"
+/>
+
+<BoxLink
+  title="Expo Notifications server-side SDK options"
+  description="Learn more about sending push notifications using a server."
+  href="/push-notifications/sending-notifications/#send-push-notifications-using-a-servers"
+/>
+
+---
+
+## OneSignal
+
+[OneSignal](https://onesignal.com/) is a customer engagement platform that provides push notifications, in-app messaging, SMS, and email services for web and mobile apps. It includes an [Expo SDK](https://github.com/OneSignal/onesignal-expo-plugin) for easy integration.
+
+<BoxLink
+  title="OneSignal Expo SDK Setup"
+  description="Follow this guide for a step-by-step setup on how to integrate OneSignal in your Expo project."
+  href="https://documentation.onesignal.com/docs/react-native-expo-sdk-setup"
+/>
+
+---
+
+## Braze
+
+[Braze](https://www.braze.com/) is a customer engagement platform that delivers personalized, cross-channel messaging through push notifications, in-app messaging, email, SMS, and web. It provides a [React Native SDK](https://github.com/braze-inc/braze-react-native-sdk) and an [Expo plugin](https://github.com/braze-inc/braze-expo-plugin/tree/main). Check out the [Expo example app](https://github.com/braze-inc/braze-expo-plugin/tree/main/example) for more details.
+
+<BoxLink
+  title="Braze Expo Setup"
+  description="Follow this guide for a step-by-step setup on how to integrate Braze in your Expo project."
+  href="https://www.braze.com/docs/developer_guide/platforms/react_native/sdk_integration"
+/>
+
+---
+
+## Customer.io
+
+[Customer.io](http://Customer.io) provides two main services:
+
+- **Journeys**, for setting up campaigns and sending messages, emails, and push notifications
+- **Data Pipelines**, for capturing and forwarding data from your Expo app, website, or other sources to the right destinations in your stack
+
+It offers an [Expo plugin](https://github.com/customerio/customerio-expo-plugin) for seamless integration.
```

Comment:
It'd be good to keep the same format as the others. Some distinctive Customer.io features we could note:
- Device-side metrics
- Invoke deep links in your app when tapping on a notification
- Define in-app messaging in addition to push notifications
- Support and guides for using customer.io notifications alongside other providers